sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -389,15 +392,48 @@ async function main() {
 main();
 ```
 
+### Dynamic Classification
+
+On the fly classification of the input documents into one or multiple categories. Assigns either one or multiple categories per document.
```

Comment:
Wondering if we should add that you don't need to train a model to use this

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/23208/commits/3d7138280ff9710d5dd8415f19d87c68e712809f